### PR TITLE
Add-text-definition-to-peripheral-sensory-neuron,-polymodal-neuron,-c…

### LINKED ISSUE #2658: Adjust definition of [columnar neuron]

### LINKED ISSUE #2659: Adjust definition of [commisural neuron]

### LINKED ISSUE #2660: Adjust definition of [polymodal neuron]

### LINKED ISSUE #2661: Adjust definition of [peripheral sensory neuron]

### PR DESCRIPTION
**Pull request by AvolaAmg**
…ommissural-neuron,-columnar-neuron

Fixes #2658
Fixes #2659
Fixes #2660
Fixes #2661

### CODE REVIEW

**Comment by AvolaAmg** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -9410,6 +9413,8 @@ SubClassOf(obo:CL_0000677 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000127
 
 # Class: obo:CL_0000678 (commissural neuron)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31693445") obo:IAO_0000115 obo:CL_0000678 "A neuron with soma location in the central nervous system that project its axon to the contralateral side of a central nervous system. This neuron can have its soma in the spinal cord dorsal root ganglion, in the hemisphere of the brain, in the retina or in the ventral nerve cord of invertebrates.")
```
[the remark this answers is not in the document]
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:9416]: ups my bad I will change the definition

[the remark this answers is not in the document]
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:9416]: technically, commissural neurons are those neurons whose axons travel contralateral though a CNS commissure in the brain to reach the opposite side of it. Retinal Ganglion Cells have their soma in the retina and their axons travel through the optic chiasm, a major brain commissure, to reach the lateral ganglionic eminence. Have a look at these references
https://www.sciencedirect.com/science/article/pii/S0012160607009098#fig3

**Comment by AvolaAmg** [comment]
#gogoeditdiff

**Comment by Caroline-99** [comment]
Everything looks good. Thank you for doing the changes

**Review by Caroline-99 — APPROVED** [review]

**Pull request merged**